website/docs/Software/OpenBCISoftware/01-OpenBCI_GUI.md

```diff
@@ -288,26 +288,42 @@ Expert Mode Keyboard Shortcuts:
+The OpenBCI GUI is built using [Processing 4](https://processing.org/), a popular, Java-based creative coding framework. If you are interested in adding features or modifying the existing code, it is really easy to do so if you are familiar with Java. If you're not familiar with Java, don't fret! Processing is one of the easiest software packages to pick up as a beginner coder.
 
 The things you will need to run the OpenBCI GUI in Processing are:
 
-- [Processing App](https://processing.org/download/?processing)
+- [Processing App](https://processing.org/download)
 - [OpenBCI GUI Sketch](https://github.com/OpenBCI/OpenBCI_GUI)
 
-First, go to processing.org and download the latest version of Processing. [Here is a walkthrough on how to install Processing on Ubuntu Linux.](http://www.artsnova.com/processing/installing-processing-ubuntu-linux-tutorial.html)
+### Installing Processing 4
 
-Go ahead and move it to your `Applications` folder, and launch the application. If this is the first time that you are running Processing, it will create what it calls it's `Sketch` folder in the following locations:
+First, go to Processing's website and [download the latest version of Processing](https://processing.org/download). For more information on Processing, you can review the official [Processing Getting Started Guide](https://processing.org/tutorials/gettingstarted) or the [Processing Documentation](https://processing.org/environment).
 
-On a Mac `Users/<user-name>/Documents/Processing`  
- On Windows `C:\Users\<user-name>\Documents\Processing`  
- On Linux `/home/<user-name>/sketchbook`
+Go ahead and move the Processing application to your `Applications` folder, and launch the application. If this is the first time that you are running Processing, it will create what it calls it's `Sketch` folder in the following locations:
 
-![GUI repo](../../assets/SoftwareImages/OpenBCISoftware/ganglion_GUI-repo.png)
+- On a Mac -`Users/<user-name>/Documents/Processing`  
+- On Windows - `C:\Users\<user-name>\Documents\Processing`  
+- On Linux -`/home/<user-name>/sketchbook`
 
-The OpenBCI GUI code repository is located on github [here](https://github.com/OpenBCI/OpenBCI_GUI). click on the `Clone or download` button in green on the right, and select `Download ZIP`. If you are a advanced GitHub user, go ahead and clone it, or fork it if you like.
+:::note
+Linux Users
+
+[Here is a walkthrough on how to install Processing on Ubuntu Linux.](http://www.artsnova.com/processing/installing-processing-ubuntu-linux-tutorial.html)
+:::
+
+:::note
+Mac Users
+
+Please use the __Intel 64-bit__ version of Processing 4 on all Mac computers. This will still run great on Apple Silicon. This allows us to continue using libraries that may not have a build for Arm64 architecture.
+:::
```

Comment:
Optional - It might be easy to miss this. Perhaps it should be an info instead of a note.

Reply:
Agreed.